examples/src/main/java/io/dapr/examples/state/StateClient.java

```diff
@@ -34,7 +34,7 @@ public String toString() {
     }
   }
 
-  private static final String STATE_STORE_NAME = "statestore";
+  private static final String STATE_STORE_NAME = "state/store";
```

Comment:
Is this intentional? `state/store` instead of `statestore`

Reply:
haha. It was a left over from another work.

Reply:
happens :)